examples/SimpleERC20/Hydra.sol

```diff
@@ -265,15 +244,15 @@ contract SimpleERC20 {
     function payBounty() private {
 
         if (DEBUG_MODE) {
-            BountyPayed();                                                      // 750 GAS
+            BountyPayed();
         }
 
         uint256 bal = this.balance - bountyValue;
         uint256 bounty = bountyValue + bal / 10;
         uint256 rest = this.balance - bounty;
 
-        bountyClaimed = true;                                                   // 20000 GAS
-        msg.sender.send(bounty);                                                //  9700 GAS
-        creator.send(rest);                                                     //  9700 GAS
+        bountyClaimed = true;
+        msg.sender.send(bounty);
+        creator.send(rest);
```

Comment:
use transfer instead of send here, to ensure that won't be funds stuck in the contract?

Reply:
agreed